<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>610</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 245);</string>
   </property>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>95</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>G마켓 산스 TTF Bold</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(101, 45, 142);
border-color: rgb(101, 45, 142);
border-style: solid;
border-width:1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>돌아가기</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>143</y>
      <width>761</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_2{
border-color: rgb(101, 45, 142);
border-style:solid;
border-width:3px;
background-color:rgb(255, 255, 255);
}

</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget#widget_4{
border-color: rgb(101, 45, 142);
border-style:solid;
border-width:3px;
background-color:rgb(255, 255, 255);
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>59</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(170, 206, 56);
background-color: rgb(255, 255, 255);
border-color: rgb(170, 206, 56);
border-style: solid;
border-width:3px;
border-radius: 10px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 206, 56);
	border-color: rgb(170, 206, 56);
	border-style:solid;
	border-width:3px;
}
QPushButton:checked{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 206, 56);
	border-color: rgb(170, 206, 56);
	border-style:solid;
	border-width:3px;
}</string>
    </property>
    <widget class="QPushButton" name="back_btn_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>방문 관광객</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="back_btn_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>35</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>인프라</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="back_btn_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>35</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>편의점</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="back_btn_5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>35</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>G마켓 산스 TTF Bold</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>결과</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>티머니 둥근바람 ExtraBold</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ㅇㅇㅇㅇㅇㅇㅇㅇㅇㅇㅇㅇ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>widget_3</zorder>
   <zorder>back_btn</zorder>
   <zorder>widget_2</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
